src/components/pages/home/events.tsx

```diff
@@ -55,179 +135,18 @@ const Events = () => {
-          />
-          <div className='relative flex flex-wrap items-center justify-between gap-2'>
-            {isOverflowing ? (
-              <Tooltip content={communityName}>
-                <div className='rounded-md border-2 border-black bg-white px-2 py-1 text-xs text-black'>
-                  <span ref={communityNameRef} className='block max-w-[200px] truncate'>
-                    {communityName}
-                  </span>
-                </div>
-              </Tooltip>
-            ) : (
-              <div className='rounded-md border-2 border-black bg-white px-2 py-1 text-xs text-black'>
-                <span ref={communityNameRef} className='block max-w-[200px] truncate'>
-                  {communityName}
-                </span>
-              </div>
-            )}
-            {logo && (
-              <Image
-                src={logo}
-                alt={`${title} logo`}
-                width={24}
-                height={24}
-                className='rounded-sm object-cover filter transition-all duration-300 hover:filter-none'
-              />
-            )}
-          </div>
-
-          <a href={link} target='_blank' rel='noopener noreferrer' className='block'>
-            <h3
-              className={`mb-2 mt-3 text-xl font-medium text-black transition-all duration-300`}
-              style={{
-                height: `${isMonthly ? monthlyCardHeight : upcomingCardHeight}px`,
-                overflow: 'hidden'
-              }}
-              title={title}
-            >
-              {title}
-            </h3>
-
-            <div className='flex-row items-center text-sm text-gray-600'>
-              <div className='flex items-center space-x-2'>
-                <span className={`rounded bg-green-100 px-2 py-0.5 text-xs text-green-800`}>
-                  {location}
-                </span>
-                <span className={`rounded bg-blue-100 px-2 py-0.5 text-xs text-blue-800`}>
-                  {date}
-                </span>
-                <AddToCalendar
-                  eventTitle={title}
-                  eventVenue={venue}
-                  eventDate={date}
-                  eventLink={link}
-                />
-              </div>
-              <div className='mt-auto flex flex-grow flex-col justify-end'>
-                <span className='mt-4 flex items-start gap-1 text-xs'>
-                  <MapPin size={16} className='mt-0.5 min-w-[16px]' />{' '}
-                  <span className='break-words'>{validateAndFormatVenue(venue)}</span>{' '}
-                </span>
-              </div>
-            </div>
-          </a>
-        </div>
-      </div>
-    );
-  };
-
   return (
     <main className='mx-4 rounded-xl bg-white p-6 md:mx-8 lg:mx-16'>
       <section>
         <h2 className='mb-3 text-lg font-normal'>
-          <span className='text-[30px] font-semibold text-black'>this month</span>
+          <span className='text-[30px] font-semibold text-black'>This Month</span>
```

Comment:
please change back to this month and upcoming, it is part of the design system of tt.